build-config/make/i2ctools.make

```diff
@@ -101,15 +99,7 @@ i2ctools-install: $(I2CTOOLS_INSTALL_STAMP)
 $(I2CTOOLS_INSTALL_STAMP): $(SYSROOT_INIT_STAMP) $(I2CTOOLS_BUILD_STAMP) $(ZLIB_INSTALL_STAMP)
 	$(Q) rm -f $@ && eval $(PROFILE_STAMP)
 	$(Q) echo "==== Installing i2ctools in $(DEV_SYSROOT) ===="
-	$(Q) PATH='$(CROSSBIN):$(PATH)'	$(MAKE) -C $(I2CTOOLS_DIR) \
-		CROSS_COMPILE=$(CROSSPREFIX) CFLAGS="$(ONIE_CFLAGS)" \
-		LDFLAGS="$(ONIE_LDFLAGS)" SYSEEPROM_ENABLE=$(I2CTOOLS_SYSEEPROM)
```

Comment:
Don't you need to leave the `make` line, otherwise onie-syseeprom will not be built on the systems that need it?

I agree the `cp` lines below can go.  I think the `make` line has to stay.

A good test would be to build a system that still uses i2ctools for onie-syseeprom and see that onie-syseeprom ends up in the sysroot.  No need to actually run the machine, but build it and see the file ends up in the sysroot correctly.

A good test machine is dell/dell_s6000_s1220.

Reply:
Yes, the same `make` line has been done in `i2ctools-build` so that I removed it from `i2ctools-install`.  I ever made `I2CTOOLS_SYSEEPROM` be enabled in AS7712_32X to see whether onie-syseeprom in i2ctools can be built or not.  But the test log was gone.

The test result on dell_s6000_s1220:

    $ ls -l dell_s6000_s1220-r0/sysroot/usr/bin/onie-syseeprom
    -rwxrwxr-x 1 david_yang david_yang 17616 Oct 13 09:31 dell_s6000_s1220-r0/sysroot/usr/bin/onie-syseeprom
    $ ls -l dell_s6000_s1220-r0/i2c-tools/i2c-tools-3.1.1/sys_eeprom/
    total 116
    -rw-rw-r-- 1 david_yang david_yang  4914 Oct 13 09:24 24cXX.c
    -rw-r--r-- 1 david_yang david_yang  2060 Oct 13 09:24 24cXX.h
    -rw-rw-r-- 1 david_yang david_yang  5808 Oct 13 09:25 24cXX.o
    -rw-rw-r-- 1 david_yang david_yang  1176 Oct 13 09:24 Module.mk
    -rwxrwxr-x 1 david_yang david_yang 21643 Oct 13 09:25 onie-syseeprom
    -rw-rw-r-- 1 david_yang david_yang 21806 Oct 13 09:24 onie_tlvinfo.c
    -rw-rw-r-- 1 david_yang david_yang  5161 Oct 13 09:24 onie_tlvinfo.h
    -rw-rw-r-- 1 david_yang david_yang 14672 Oct 13 09:25 onie_tlvinfo.o
    -rw-rw-r-- 1 david_yang david_yang  4150 Oct 13 09:24 sys_eeprom.c
    -rw-rw-r-- 1 david_yang david_yang  7352 Oct 13 09:25 sys_eeprom.o
    -rw-rw-r-- 1 david_yang david_yang   311 Oct 13 09:24 sys_eeprom_platform.h

Reply:
BTW, the test result on AS7712_32X:

    $ ls -l accton_as7712_32x-r0/sysroot/usr/sbin/i2c*
    lrwxrwxrwx 1 david_yang david_yang 17 Oct 12 13:47 accton_as7712_32x-r0/sysroot/usr/sbin/i2cdetect -> ../../bin/busybox
    lrwxrwxrwx 1 david_yang david_yang 17 Oct 12 13:47 accton_as7712_32x-r0/sysroot/usr/sbin/i2cdump -> ../../bin/busybox
    lrwxrwxrwx 1 david_yang david_yang 17 Oct 12 13:47 accton_as7712_32x-r0/sysroot/usr/sbin/i2cget -> ../../bin/busybox
    lrwxrwxrwx 1 david_yang david_yang 17 Oct 12 13:47 accton_as7712_32x-r0/sysroot/usr/sbin/i2cset -> ../../bin/busybox

Reply:
great!  thanks for the info